data/iatemplates/mongodb_too_many_connections.yml

```diff
@@ -0,0 +1,27 @@
+---
+templates:
+  - name: mongodb_too_many_connections
```

Comment:
looks like wrong name for this alert,isn't it?

Reply:
I have renamed it.